packages/core/project.json

```diff
@@ -32,7 +32,7 @@
       "outputs": ["{options.outputPath}"],
       "options": {
         "uiFramework": "@storybook/html",
-        "outputPath": "dist/storybook/core",
+        "outputPath": "packages/core/dist",
```

Comment:
This should be inside of the `packages`

nx convention is to use `dist/[package-name]/ ... `

Reply:
reverted

Reply:
Thanks!